TombEngine/Game/StaticObject.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "Math/Math.h"
+#include "Objects/objectslist.h"
+
+using namespace TEN::Math;
+
+enum GAME_OBJECT_ID : short;
+
+// TODO: Come up with a namespace.
+//namespace TEN
+//{
+	enum StaticMeshFlags
+	{
+		SM_VISIBLE = 1 << 0,
+		SM_SOLID   = 1 << 1
+	};
+
+	// class StaticObject
+	struct MESH_INFO
+	{
+	public:
+		// Fields
+
+		std::string	   Name		= {};
+		//int			   Id		= 0; // TODO: Store static ID.
+		GAME_OBJECT_ID ObjectId = GAME_OBJECT_ID::ID_NO_OBJECT;
```

Comment:
`GAME_OBJECT_ID` enumeration exists only for moveables. Statics use only a slot index, there is no unique properties for different static mesh slots.